### ISSUE TITLE
Reading a Notification should dismiss the notification

### ISSUE THREAD

**Issue by niftynei** [zulip/zulip-android#209]
If I read a message that I received a notification about, the notification should be dismissed.

**Comment by AmoliShah** [comment]
I would like to work on this issue. @niftynei

**Comment by timabbott** [comment]
Go for it @AmoliShah!

**Comment by AmoliShah** [comment]
@timabbott Changes are to be made in RecyclerMessageAdapter.java activity. Right?

**Comment by saketkumar95** [comment]
@AmoliShah No. it is for gsm. [Check it out here](https://github.com/zulip/zulip-android/tree/master/app/src/main/java/com/zulip/android/gcm)

**Comment by AmoliShah** [comment]
ok thanks. @saketkumar95

**Comment by saketkumar95** [comment]
@AmoliShah Are you working on this?
